mongo/collection.go

```diff
@@ -561,7 +619,18 @@ func (coll *Collection) UpdateMany(ctx context.Context, filter interface{}, upda
 	return coll.updateOrReplace(ctx, f, update, true, rrMany, true, opts...)
 }
 
-// ReplaceOne replaces a single document in the collection.
+// ReplaceOne executes an update operation (https://docs.mongodb.com/manual/reference/command/update/) to replace at
+// most one document in the collection.
+//
+// The filter parameter must be a document containing query operators and can be used to select the document to be
+// replaced. It cannot be nil. An empty document (e.g. bson.D{}) will select one of the documents in the collection. If
+// the filter does not match any documents, the operation will succeed and an UpdateResult with a MatchedCount of 0 will
```

Comment:
> the operation will succeed

Will it be obvious to Go developers what this means?  i.e, that the method will return with error == nil?

Reply:
I think a Go user would be able to understand what success means.